### DISCUSSION TITLE
The ot-cli-ftd file size is 608KB,  it exceeds cc2538 Flash size of 512KB. How can I solve this problem?

### DISCUSSION THREAD

**Discussion by imlsq** [openthread/openthread#7794] [Q&A] [answered]
I clone cc2538 code from https://github.com/openthread/ot-cc2538.git
compile
./script/build
The ot-cli-ftd file size is 608KB,  it exceeds the cc2538 Flash size of 512KB. How can I solve this problem?

du -b build/bin/ot-cli-ftd
623612  build/bin/ot-cli-ftd

**Comment by jwhui** [comment, marked as answer]
The `build/bin/ot-cli-ftd` file is an `elf` file, not the actual binary flashed to the device.

See the [Flash Binaries section in the CC2538 README](https://github.com/openthread/ot-cc2538#flash-binaries) for more details.
